index.js

```diff
@@ -90,12 +97,26 @@ client.on('message', async (channel, tags, message, self) => {
             } else {
                 await handleSongRequest(channel, tags[displayNameTag], message, true);
             }
-    } else if (messageToLower === chatbotConfig.skip_alias) {
+    } else if (chatbotConfig.allow_volume_set && messageToLower.split(" ")[0] == '!volume') {
```

Comment:
Don't hardcode commands, these should be configurable in the config file. I'll also be fixing `!song` to behave like this.

Reply:
!song is a previously established command and therefore we are following previously established coding standard.
![image](https://user-images.githubusercontent.com/22343767/201497004-2f254c8d-5115-4e3f-a32b-5e8f802d6c93.png)